docs/core/setup/dependencies.md

```diff
@@ -0,0 +1,266 @@
+
+### Ubuntu
+
+Ubuntu distributions require the following libraries installed:
+
+- liblttng-ust0
+- libcurl3 (for 14.x and 16.x)
+- libcurl4 (for 18.x)
+- libssl1.0.0
+- libkrb5-3
+- zlib1g
+- libicu52 (for 14.x)
+- libicu55 (for 16.x)
+- libicu57 (for 17.x)
+- libicu60 (for 18.x)
+
+For .NET Core apps that use the *System.Drawing.Common* assembly, you also need the following dependency:
+
+- libgdiplus (version 6.0.1 or later)
+
+> [!WARNING]
+> Most versions of Ubuntu include an earlier version of libgdiplus. You can install a recent version
+> of libgdiplus by adding the Mono repository to your system. For more information,
+> see <https://www.mono-project.com/download/stable/>.
+
+### CentOS and Fedora
+
+CentOS distributions require the following libraries installed:
+
+- lttng-ust
+- libcurl
+- openssl-libs
+- krb5-libs
+- libicu
+- zlib
+
+Fedora users: If your openssl's version >= 1.1, you'll need to install **compat-openssl10**.
+
+For .NET Core 2.0, following dependencies are also required:
+
+- libunwind
+- libuuid
+
+For more information about the dependencies, see [Self-contained Linux apps](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md).
+
+For .NET Core apps that use the *System.Drawing.Common* assembly, you'll also need the following dependency:
+
+- libgdiplus (version 6.0.1 or later)
+
+> [!WARNING]
+> Most versions of CentOS and Fedora include an earlier version of libgdiplus. You can install a recent version
+> of libgdiplus by adding the Mono repository to your system. For more information,
+> see <https://www.mono-project.com/download/stable/>.
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+.NET Core is supported on the following macOS releases:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| .NET Core Version | macOS                 | Architectures |     |
+| ----------------- | --------------------- | --------------| --- |
+| 3.0               | High Sierra (10.13+)  | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md) |
+| 2.2               | Sierra (10.12+)       | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md) |
+| 2.1               | Sierra (10.12+)       | x64 | [More information](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md) |
+
+## libgdiplus
+
+.NET Core applications that use the *System.Drawing.Common* assembly require libgdiplus to be installed.
+
+An easy way to obtain libgdiplus is by using the [Homebrew ("brew")](https://brew.sh/) package manager for macOS. After installing *brew*, install libgdiplus by executing the following commands at a Terminal (command) prompt:
+
+```console
+brew update
+brew install mono-libgdiplus
+```
+
+::: zone-end
+
+## Next steps
+
+- To develop and run apps, install the [.NET Core SDK](install-sdk.md). (includes the runtime)
```

Comment:
```suggestion
- To develop and run apps, install the [.NET Core SDK](install-sdk.md) (includes the runtime).
```

Reply:
Should we use the "Next steps" button for any next steps links? Like this: https://docs.microsoft.com/en-us/visualstudio/get-started/csharp/tutorial-console?view=vs-2019#next-steps

> [!div class="nextstepaction"]
> [Install .NET Core runtime](install-runtime.md)